identity-service/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@amplitude/node": "^1.9.2",
-    "@audius/sdk": "1.0.37",
+    "@audius/sdk": "2.0.3-beta.4",
```

Comment:
there's no breaking API changes we need to adjust for in this release?

Reply:
not that im aware of :)